lib/nomad/api/job.rb

```diff
@@ -11,6 +11,48 @@ def job
   end
 
   class Job < Request
+    # Dispatch parametrized job
+    #
+    # @example
+    #   Nomad.job.dispatch 'task/uuid-uuid-uuid', 'blabla', {
+    #
+    #   } #=> #<JobDispatch ...>
+    #
+    # @option [String] :name
+    #   dispatch job based on the given name
+    #
+    # @option [String] :payload_contents
+    #   dispatch job payload file content
+    #
+    # @option [Hash] :meta
+    #   dispatch job meta parameters
+    #
+    # @return [JobDispatch]
+    def dispatch(name, payload_contents, meta, **options)
```

Comment:
Do you mind changing `payload_contents` to `contents` to match create?

Reply:
It was long time ago) Currently I will prefer Golang integration over Ruby one